Plate.py

```diff
@@ -0,0 +1,159 @@
+class Plate:
+    """
+    A class to represent a plate.
+    ...
+    Attributes
+    ----------
+    diameter : float
+        plate diameter in centimeters.
+    material : str
+        material of the plate.
+    color : str
+        color of the plate.
+    is_clean : bool
+        condition of the plate (clean/dirty).
+    has_food : bool
+        availability of food.
+    Methods
+    -------
+    wash():
+        cleans the plate and sets the isClean flag to True.
+    dirty():
+        marks the plate as dirty (sets the is_clean flag to False).
+    eat():
+        resets the has_food flag and marks the plate as dirty.
+    has_food():
+        sets the has_food flag to true
+    """
+    instance = None
+
+    def __init__(self, diameter=None, material=None, color=None, is_clean=False, has_food=False):
+        self.__diameter = diameter
+        self.__material = material
+        self.__color = color
+        self.__is_clean = is_clean
+        self.__has_food = has_food
+
+    def wash(self):
+        """
+        Cleans the plate and sets the is_clean flag to True.
+
+        Parameters
+        ----------
+        None
+
+        Return
+        ----------
+        None
+        """
+        self.__is_clean = True
+
+    def dirty(self):
```

Comment:
Renamet this method. According to convention methods must start with verb. Me personally, at first I thought it would be some kind of boolean field getter, which is not. So make it more readable

Reply:
such method names are provided by the teacher